client/src/components/Monument/Gallery/Gallery.js

```diff
@@ -168,7 +168,7 @@ export default class Gallery extends React.Component {
                         <div className="d-flex justify-content-center">
                             <img src={selectedImage.url} alt="large"/>
                         </div>
-                        <p className="caption">Image description...</p>
+                        {/*<p className="caption">Image description...</p>*/}
```

Comment:
Leftover code or something?

Reply:
Yea, I noticed this was laying around and we never actually added captions to images yet so for now I'm hiding it

Reply:
Ah ok didn't realize that.